utils/getDuplicateSections/index.test.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const getDupicateSections = require('./index');
+
+describe('getDuplicateSections', () => {
+    describe('Given sections from multiple tasks', () => {
+        it('should return any sectionIds that appear is multiple tasks', () => {
```

Comment:
should this be "appear _as_ multiple tasks?"

Reply:
It was supposed to be "appear _in_ multiple tasks?"